plugins/aws_k8s/resoto_plugin_aws_k8s/__init__.py

```diff
@@ -0,0 +1,44 @@
+from resotolib.baseplugin import BasePostCollectPlugin
+from resotolib.baseresources import BaseResource
+from resotolib.graph import BySearchCriteria, ByNodeId, Graph
+
+from resotolib.logger import log
+import functools
+from typing import Any
+
+
+def rgetattr(obj: Any, attr: str, *args: Any) -> Any:
+    def _getattr(obj: Any, attr: str) -> Any:
+        return getattr(obj, attr, *args)
+
+    return functools.reduce(_getattr, [obj] + attr.split("."))
+
+
+def link_k8s_node_to_aws_nodegroup(graph: Graph, resource: BaseResource) -> None:
+    if resource.kind == "kubernetes_node":
+        if (labels := rgetattr(resource, "labels", {})) and (nodegroup := labels.get("eks.amazonaws.com/nodegroup")):
+            graph.add_deferred_edge(
+                BySearchCriteria(f"is(aws_eks_nodegroup) and reported.id={nodegroup}"),
+                ByNodeId(resource.chksum),
+            )
+
+
+def link_k8s_cluster_to_eks_cluster(graph: Graph, resource: BaseResource) -> None:
+    if resource.kind == "kubernetes_cluster":
+        graph.add_deferred_edge(
+            BySearchCriteria(f"is(aws_eks_cluster) and reported.arn={resource.id}"),
```

Comment:
`arn`==`id` sounds wrong to me.

Reply:
that's how it is collected.

Reply:
Yes this is coming from the k8s side - it does not know about arns. This is fine.